src/js/edu-benefits/pages/schoolSelection.js

```diff
@@ -66,12 +66,30 @@ export default function createSchoolSelectionPage(schema, options) {
     civilianBenefitsAssistance: {
       'ui:title': civilianBenefitsLabel,
       'ui:widget': 'yesNo'
+    },
+    currentlyActiveDuty: {
+      nonVaAssistance: {
```

Comment:
Will this get merged into the `currentlyActiveDuty` object from the `militaryService` chapter?

Reply:
Yep